### DISCUSSION TITLE
expired artifact (debug)

### DISCUSSION THREAD

**Discussion by miyuhub** [JingMatrix/LSPatch#23] [General]
please kindly update dl links, i'd like to download debug specifically, thank you!
https://nightly.link/JingMatrix/LSPatch/actions/runs/10888848425

**Comment by JingMatrix** [comment]
Hello! You can find latest debug build from GitHub Actions, for example, this one: https://github.com/JingMatrix/LSPatch/actions/runs/12103943429
